packages/manager/modules/manager-pci-common/src/api/data/regions.ts

```diff
@@ -1,10 +1,16 @@
 import { fetchIcebergV6 } from '@ovh-ux/manager-core-api';
 
-export type TRegion = {
+export type TRegionType = 'region' | 'localzone' | 'region-3-az' | string;
+
+export type TRegionBase = {
   name: string;
-  type: 'region' | 'localzone' | 'region-3-az' | string;
+  type: TRegionType;
+  availabilityZones: string[];
+  continentCode: 'ASIA' | 'EU' | 'NA' | 'US' | string;
+};
+
+export type TRegion = TRegionBase & {
```

Comment:
A bit strange to split these two types, as a `Region` is made of all these properties.
Maybe we could consider using `Omit` to create a `TProductAvailabilityRegion`

Reply:
The best would be to rename TRegion but would cause a breaking change

Omit isn't a good idea since some of these properties are generated by backend and not related to the object itself, Pick could do the trick also